src/frontend/src/styles/main.css

```diff
@@ -651,3 +651,7 @@ nav {
 button[data-action="protect"] {
   margin-bottom: 0;
 }
+
+input {
+  line-height: 1em;
```

Comment:
is this related to the PR?

Reply:
Yes it is, the input line-height decreases by default when the input gets contained inside <form>. 
I created a custom class name, and applied styling to just this specific input [here](https://github.com/dfinity/internet-identity/pull/775/commits/ee92e6ceb80895529cca024296d53645e124e5ef)